### ISSUE TITLE
assert.isFalse(foo) transformed to expect(foo).toBe()

### ISSUE THREAD

**Issue by tschaub** [skovhus/jest-codemods#20]
I have `example.js` with the following:

```js
const assert = require('chai').assert;

const foo = false;
assert.isFalse(foo);
```

After running `jest-codemods example.js`, it looks like this:

```js
const foo = false;
expect(foo).toBe();
```

Where before no error would be thrown, after the change, Jest fails with this:

```
Error: expect(received).toBe(expected)

Expected value to be (using ===):
  undefined
Received:
  false
```

**Comment by skovhus** [comment]
Thanks for reporting this! And fixing it. I'll have a look.

**Comment by skovhus** [comment]
Released as 0.5.1 (thanks again for reporting this!)

Let me know if it doesn't work for you.
